street-explorer/www/js/layers.js

```diff
@@ -49,7 +49,7 @@ export const makePlainGeoJsonLayer = (text) => {
   });
 };
 
-export const makeLanePolygonLayer = (text) => {
+export const lanePolygonStyle = (feature) => {
```

Comment:
Just splitting this out so we can render lanes without getting the popup behavior